api/comments.rb

```diff
@@ -1,3 +1,33 @@
+get "#{APIPREFIX}/comments" do
```

Comment:
rather than passing user-id as a query string, can you please pass user as part of the URL param?
`get "#{APIPREFIX}/users/:user_id/comments" do |user_id|`

Reply:
I think that would tie this API too much to users. It's possible in the future it might need to filter by other parameters.

Reply:
right, thanks for explaining the future intent.

Reply:
> I think that would tie this API too much to users. It's possible in the future it might need to filter by other parameters.

Hey @xitij2000 @awaisdar001, please make sure this sort of thinking is captured via comments in the code itself, and not just review comments in the PR. Future developers generally won't have time to look up PR comments to figure out why things were done a certain way.

Reply:
Good point. thanks for brining it up @bradenmacdonald

Reply:
@awaisdar001 I added a comment clarifying the intent, as suggested by @bradenmacdonald